### PR TITLE
Add image-meta-tag as new recipe

### PR DESCRIPTION
**Pull request by malcolmbrooks**
Adds a recipe for python package image-meta-tag at vn0.6.3

### CODE REVIEW

File: recipes/image-meta-tag/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  fn: {{ version }}.tar.gz
+  url: {{ giturl }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+      - python >=2.7,<3
```
Comment (synapticarbors) [inline recipes/image-meta-tag/meta.yaml]: Just use `python` here without versions and then put:

```
skip: True  # [py3k]
```

in the `build` section above.

File: recipes/image-meta-tag/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  version:  {{ version }}
+
+source:
+  git_url: {{ giturl }}
+  git_tag: {{ version }}
+  fn: {{ version }}.tar.gz
+  url: {{ giturl }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/image-meta-tag/meta.yaml]: add:

```
skip: True  # [py3k]
script: python setup.py install --single-version-externally-managed --record record.txt
```

File: recipes/image-meta-tag/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+package:
+  name: image-meta-tag
+  version:  {{ version }}
+
+source:
+  git_url: {{ giturl }}
+  git_tag: {{ version }}
+  fn: {{ version }}.tar.gz
+  url: {{ giturl }}/{{ name }}/archive/{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/image-meta-tag/meta.yaml]: I think `{{name}}` should be here. The link I'm seeing is:

https://github.com/SciTools-incubator/image-meta-tag/archive/0.6.3.tar.gz

File: recipes/image-meta-tag/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+build:
+  number: 0
+
+requirements:
+  build:
+      - python >=2.7,<3
+      - setuptools
+  run:
+      - python >=2.7,<3
```
Comment (synapticarbors) [inline recipes/image-meta-tag/meta.yaml]: Just use `python` here.

File: recipes/image-meta-tag/meta.yaml (hunk 3 above)
Comment (malcolmbrooks) [reply recipes/image-meta-tag/meta.yaml]: Thanks, you're right, the {{ name }} is superfluous as it's in the {{ giturl }}.

File: recipes/image-meta-tag/meta.yaml (hunk 2 above)
Comment (malcolmbrooks) [reply recipes/image-meta-tag/meta.yaml]: Thanks, I'd omitted the script line for the build, which is rather important.

File: recipes/image-meta-tag/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+{% set version = "0.6.4" %}
+{% set giturl = "https://github.com/SciTools-incubator/image-meta-tag" %}
+{% set sha256 = "4edb88ad364e03642b502d6dbf68f15cc7533d6c7d0d8db0619a240e9134a128" %}
+
+package:
+  name: image-meta-tag
+  version:  {{ version }}
+
+source:
+  git_url: {{ giturl }}
```
Comment (synapticarbors) [inline recipes/image-meta-tag/meta.yaml]: I think you want to get rid of the `git_url` and `git_tag` here.

**Comment by malcolmbrooks** [comment]
Thanks @synapticarbors, I was just trying that exact thing.

**Comment by synapticarbors** [comment]
It looks like there is a windows specific issue upstream in the source `setup.py`.

**Comment by synapticarbors** [comment]
I think the following:

https://github.com/SciTools-incubator/image-meta-tag/blob/master/setup.py#L37

```
data_files = [('ImageMetaTag/javascript/', glob('ImageMetaTag/javascript/*'))],
```

can just be changed to:

```
data_files = [('ImageMetaTag/javascript', glob('ImageMetaTag/javascript/*'))],
```

but I'm not 100% sure.

**Comment by malcolmbrooks** [comment]
Hi @synapticarbors 
this is now passing all tests. What happens next?

Thanks

**Comment by synapticarbors** [comment]
@malcolmbrooks  -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**